File: app/model/QualityRatingAffectedOrderDTO.scala
package model

import play.api.libs.json._

/**
  * Информация о заказе, который повлиял на индекс качества.
  * @param orderId Идентификатор заказа.
  * @param description Описание проблемы.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2024-09-09T22:33:09.055496365Z[Etc/UTC]", comments = "Generator version: 7.8.0")
case class QualityRatingAffectedOrderDTO(
  orderId: Long,
  description: String,
  componentType: AffectedOrderQualityRatingComponentType
)

object QualityRatingAffectedOrderDTO {
  implicit lazy val qualityRatingAffectedOrderDTOJsonFormat: Format[QualityRatingAffectedOrderDTO] = Json.format[QualityRatingAffectedOrderDTO]
}

File: ympa_fsharp_giraffe_server/src/model/QualityRatingAffectedOrderDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.AffectedOrderQualityRatingComponentType

module QualityRatingAffectedOrderDTO =

  //#region QualityRatingAffectedOrderDTO


  type QualityRatingAffectedOrderDTO = {
    OrderId : int64;
    Description : string;
    ComponentType : AffectedOrderQualityRatingComponentType;
  }
  //#endregion
